apps/condo/domains/billing/utils/serverSchema/index.js

```diff
@@ -69,15 +70,39 @@ async function registerBillingReceipts (context, data) {
  * @param period {string}
  * @return {Promise<*>}
  */
-const getPaymentsSum = async (context, organizationId, accountNumber, period, bic, bankAccount) => {
-    const payments = await  find('Payment', {
-        organization: { id: organizationId },
-        accountNumber: accountNumber,
-        period: period,
-        status_in: [PAYMENT_DONE_STATUS, PAYMENT_WITHDRAWN_STATUS],
-        recipientBic: bic,
-        recipientBankAccount: bankAccount,
-    })
+const getPaymentsSum = async (context, receiptId, organizationId, accountNumber, period, bic, bankAccount) => {
+    const directReceiptIdWhere = { AND: [
+        { receipt: { id: receiptId } },
+        { receipt_is_null: false },
+    ],
+    }
+
+    // TODO: create task to connect receipt to early payment after receipt creation
+    // TODO: remove this check after upper todo
```

Comment:
you need to add task here

format is: // TODO(YEgorLu): DOMA-NNNN what you want to do